src/components/layout/SiteLayout.tsx

```diff
@@ -1,6 +1,9 @@
 import React from "react";
 import Link from "next/link";
 import Head from "next/head";
+import { faEnvelope } from "@fortawesome/free-solid-svg-icons/faEnvelope";
```

Comment:
I think these extra imports are left over from some work in progress you were doing. These two imports (from `@fortawesome/...`) aren't actually used being used in this file, so they should be removed.

Reply:
Done! Do I need to create another pull request for the removed lines? I just staged a commit for them. 

I'm not sure what's going on with my files. When I cloned the repo for the conference site, I ended up with an extra copy of conference-site-main alongside conference-site...

Reply:
You can just push your existing commit to the branch. So you should:
* Make sure you're on this branch (`adding-buttons`)
* Push

Reply:
Got it, thank you! Hopefully I did that right